index.html

```diff
@@ -222,7 +222,83 @@ <h3 class="work-description">
                     </li>
                 </ul>
                 <div class="action">
-                    <a href="#" class="action-btn btn"><span>See Project</span></a>
+                    <a href="#" data-target="#work-modal" class="action-btn btn"><span>See Project</span></a>
+                </div>
+            </div>
+        </div>
+        <!-- Modal -->
+        <div class="popup" id="work-modal">
+            <div class="modal">
+                <div class="modal-header">
+                    <div>
+                        <h2>Tonic</h2>
+                        <div class="frame">
+                            <div class="client">
+                                <h3>Canopy</h3>
+                            </div>
+                            <div class="counter1"></div>
+                            <div class="role">
+                                <h3>Back End Dev</h3>
+                            </div>
+                            <div class="counter2"></div>
+                            <div class="year">
+                                <h3>2015</h3>
+                            </div>
+                        </div>
+                    </div>
+                    <img src="./asset/images/close-icon-dark.png" height="14" width="14" class="modal-close"
+                        alt="Close Modal Icon">
+                </div>
+                <div class="modal-body">
+                    <img class="desktop-img" src="./asset/images/Modal-Image-Desktop.png" alt="Modal Image">
+                    <img class="mobile-img" src="./asset/images/Modal-Image-Mobile.png" alt="Modal Image">
```

Comment:
- [ ] Kindly fix the overflow on the modal pop up  for the mobile version. You could add `overflow: hidden` on the x-axis or change the height of the element causing this.

![image](https://user-images.githubusercontent.com/81932478/179824177-e7fe13fc-c64f-4856-93c8-a02503d31a68.png)

Reply:
There is no overflow in the code even after using various mobile screen sizes. However, I have set overflow-x to hidden. Thanks for your review

![image](https://user-images.githubusercontent.com/48407831/179846280-9c0003ce-5122-4ec9-a688-f272b2d4620b.png)